internal/postprocessor/README.md

```diff
@@ -19,4 +19,8 @@ First, build the Docker container must be built locally and name it `postprocess
     ```
     - To test that files are being copied and processed correctly, make changes in the temporary `owl-bot-staging/` directory before running the post-processor.
 
-At this stage of development you should see files being copied into their corresponding directories at the root `google-cloud-go` with some linting.
+## Testing the post-processor locally
+From the `google-cloud-go/internal/postprocessor` directory run: 
+```sh
+go run main.go -stage-dir="../../owl-bot-staging/src/" -client-root="../.." -googleapis-dir="/home/guadriana/developer/googleapis" -testing
```

Comment:
Did we find a way to fully remove the testing flag?

Reply:
Yes, done.